hw09-jdbc/src/main/java/ru/otus/hw09/jdbc/JDBCTemplateImp.java

```diff
@@ -0,0 +1,58 @@
+package ru.otus.hw09.jdbc;
+
+import org.slf4j.LoggerFactory;
+import ru.otus.hw09.MyException;
+import ru.otus.hw09.service.DbExecutor;
+import ru.otus.hw09.sessionmanager.SessionManager;
+import java.sql.*;
+
+
+public class JDBCTemplateImp<T> implements JDBCTemplate<T> {
+
+    private SessionManager connectionManager;
+    private DbExecutor dbExecutor;
+
+    public JDBCTemplateImp(SessionManager connectionManager, DbExecutor dbExecutor) {
+        this.connectionManager = connectionManager;
+        this.dbExecutor = dbExecutor;
+    }
+
+    @Override
+    public void createTable(Class<T> clazz) {
+            dbExecutor.create(connectionManager.getConnection(),clazz);
+    }
+
+    @Override
+    public Object create(T objectData)  {
+        try {
+            return dbExecutor.insert(connectionManager.getConnection(),objectData);
+        }catch (SQLException e) {
+            throw new MyException(e.getMessage(), e.getCause());
```

Comment:
```suggestion
            throw new MyException(e);
```
Чтобы сохранялся стек вызовов.

Еще ссылки:
https://docs.oracle.com/javase/tutorial/essential/exceptions/chained.html
https://stackoverflow.com/questions/5020876/what-is-the-advantage-of-chained-exceptions